test/unit/lib/whitehall/asset_manager_storage_test.rb

```diff
@@ -186,6 +185,12 @@ def assets_protected?
 
       @uploader.store!(@file)
     end
+
+    test "should call deleteAssetWorker with asset manager id" do
+      model = create(:image_with_asset)
+      AssetManagerDeleteAssetWorker.expects(:perform_async).times(7).with(nil, regexp_matches(/asset_manager_id./))
+      model.destroy!
```

Comment:
Personal preference for some spacing here 🫠

Reply:
accepted